clean_project/clean_file_test.py

```diff
@@ -7,29 +7,33 @@
 
 class CleanFileTest(unittest.TestCase):
```

Comment:
please add some more tests around hyphens, underscores, apostrophes, double quotes and their respective utf8 variants. not something too exhaustive, but just a few test cases